internal/core/items.rs

```diff
@@ -433,47 +432,35 @@ pub struct TouchArea {
     pub cached_rendering_data: CachedRenderingData,
     /// true when we are currently grabbing the mouse
     grabbed: Cell<bool>,
-    clicked_timer: Timer,
+    // U64::MAX when unset. This is accessed through FFI, so Option is not an option
+    last_click_time_stamp: Cell<crate::animations::Instant>,
```

Comment:
Why do you use that state instead of using the `click_count` from the event?

Reply:
This is quite unfortunate.  That means that the click_count should be fixed. It's quite annoying to have two ways to check this double click and twice the states.
I guess we can already merge this and cleanup later though.